adapter/internal/oasparser/config_generator.go

```diff
@@ -311,3 +311,19 @@ func generateRPCEndpointCluster(inputEndpointCluster *model.EndpointCluster) *ap
 	}
 	return endpoints
 }
+
+func generateRPCEndpointSecurity(inputEndpointSecurity *model.EndpointSecurity) *api.EndpointSecurity {
+	if inputEndpointSecurity == nil {
+		return nil
+	}
+	var security *api.EndpointSecurity
+	if inputEndpointSecurity != nil {
+		security = &api.EndpointSecurity{
+			Type:     inputEndpointSecurity.Type,
+			Username: inputEndpointSecurity.Username,
+			Password: inputEndpointSecurity.Password,
+			Enabled:  inputEndpointSecurity.Enabled,
```

Comment:
add customParameters too, otherwise it will break backend JWT generation flow. Also, check whether this PR affects the backend JWT generation

Reply:
resolved